src/main/scala/viper/gobra/translator/encodings/maps/MapEncoding.scala

```diff
@@ -111,6 +127,50 @@ class MapEncoding extends LeafTypeEncoding {
     }
   }
 
+  /**
+    * Encodes expressions when they occur as the top-level expression in a trigger.
+    * Notice that if we had used the expression encoding for the following triggers,
+    * they would have resulted in ill-formed triggers at the Viper level (e.g., because
+    * they would have ternary operations).
```

Comment:
```suggestion
    * Encodes expressions when they occur as the top-level expression in a trigger.
    * Notice that using the expression encoding for the following triggers,
    * results in ill-formed triggers at the Viper level (e.g., because
    * they have ternary operations).
```